### PR TITLE
{geo}[foss/2022a,gompi/2022a,system/system] CliMetLab v0.12.6, Magics v4.13.0, ecBuild v3.7.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/e/ecBuild/ecBuild-3.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        'source_urls': [GITHUB_SOURCE],
+        'filename': '%(version)s.tar.gz',
+        'extract_cmd': 'tar -xzf %s --strip-components=1',
+    },
+]
+checksums = ['44ba49b6b628e88bc0145f0cd1fdbdff37fbff2327c581f22a47cd10263a21b6']
+
+buildininstalldir = True
+
+skipsteps = ['install']
```
Comment (lexming) [inline easybuild/easyconfigs/e/ecBuild/ecBuild-3.7.0.eb:26]: I'm curious what is this doing. `Tarball` will already copy the sources in the build directroy to install directory, isn't it?

File: easybuild/easyconfigs/c/CliMetLab/CliMetLab-0.12.6-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    ('PyYAML', '6.0'),
+    ('Magics', '4.13.0'),
+    ('cdsapi', '0.5.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+# stick to termcolor 1.x, to avoid hatchling required dependency
+# remove ecmwflibs requirement (provides ecCodes and Magics)
```
Comment (lexming) [inline easybuild/easyconfigs/c/CliMetLab/CliMetLab-0.12.6-foss-2022a.eb:34]: This was confusing at first, I suggest to expand this comment:
```suggestion
# remove ecmwflibs requirement as it just bundles ecCodes and Magics and there are no direct imports to it
```

File: easybuild/easyconfigs/e/ecBuild/ecBuild-3.7.0.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/e/ecBuild/ecBuild-3.7.0.eb:26]: `buildininstalldir = True` will unpack the sources directly into the installdir, so there is no need for the extra install step.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @smoors !

**Pull request merged**